Add Keymanager token

MartinNikov opened the pull request.

MartinNikov committed:
Add Keymanager token

MartinNikov committed:
Exit with non zero code when env vars are invalid/missing

zah commented on index.js:
```diff
@@ -174,12 +179,14 @@ async function main() {
 		const keymanagerUrls = params.keymanagerUrls.split(",").map(s => s.trim());
 		for (let i = 0; i < keymanagerUrls.length; i++) {
 			const keymanagerUrl = keymanagerUrls[i];
+			const token = fs.readFileSync(params.keymanagerTokenFile, 'utf-8');
```
Make sure the user is getting a friendly error message when an invalid toke file is provided or if there are permission issues with accessing the file, etc.

MartinNikov replied:
The changes I made are in the validation of a variable to have the following scenarios:
* If the variable is not set, it will give the following error: "KEYMANAGER_TOKEN_FILE is not defined.".
* If the file exists but is empty, it will give the following error: "Token file is empty."
* If the file does not exist or cannot be accessed, it will give the following error: "The token file cannot be found or does not have access rights. Please enter a valid file path."

MartinNikov committed:
fix keymanagerAPI Response schema

MartinNikov committed:
Add `OVERWRITE` environment variable

This variable can be used to indicate if you wish to overwrite
signatures without utilizing the UI.

MartinNikov committed:
Make validation of certains env variables optional

* If `KEYMANAGER_URLS` is defined, dont't require the `REMOTE_SIGNER_URL` var.
* If `KEYMANAGER_URLS` is defined, require the `KEYMANAGER_TOKEN_FILE` var.
* If `REMOTE_SIGNER_URL` is defined, dont't require the `KEYMANAGER_URLS` var.

zah commented on src/withdrawal/keymanagerAPIMessages.js:
```diff
@@ -38,14 +39,15 @@ async function requestValidatorSignature(keymanagerUrl, body) {
 
-	if(!response.data || !response.data.signature || response.data.signature.length !== 194){
+	if(!response.data.data || !response.data.data.signature || response.data.data.signature.length !== 194){
+		console.log (response);
```
```suggestion
```

zah committed:
Update src/withdrawal/keymanagerAPIMessages.js
